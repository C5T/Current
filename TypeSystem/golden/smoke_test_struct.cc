../Schema/golden/smoke_test_struct.cc