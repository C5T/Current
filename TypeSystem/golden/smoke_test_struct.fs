../Schema/golden/smoke_test_struct.fs