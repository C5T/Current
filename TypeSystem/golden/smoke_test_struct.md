../Schema/golden/smoke_test_struct.md