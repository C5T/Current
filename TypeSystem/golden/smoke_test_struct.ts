../Schema/golden/smoke_test_struct.ts