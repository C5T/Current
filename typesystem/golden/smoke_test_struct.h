../Schema/golden/smoke_test_struct.h