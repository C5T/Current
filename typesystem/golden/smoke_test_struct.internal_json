../schema/golden/smoke_test_struct.internal_json