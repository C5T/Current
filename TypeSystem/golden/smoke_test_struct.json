../Schema/golden/smoke_test_struct.json